Modules/_lzmamodule.c

```diff
@@ -1591,6 +1611,15 @@ lzma_exec(PyObject *module)
         return -1;
     }
 
+    if (PyModule_Add(module, "LZMA_VERSION",
+                     PyUnicode_FromString(LZMA_VERSION_STRING)) < 0) {
+        return -1;
+    }
+    if (PyModule_Add(module, "LZMA_RUNTIME_VERSION",
```

Comment:
That these are strings is unfortunate, that makes comparisons difficult.  This is seen in test_lzma.py where it has to implement its own version number parsing logic.

All users who want to understand the lzma version numbers are going to need to do the same.

Reply:
I think I'll move that code from test_lzma.py into lzma.py so that these show up as tuples to the user.